apps/wallet-mobile/src/features/Swap/common/helpers.ts

```diff
@@ -95,3 +97,30 @@ export const sortTokensByName = (a: Balance.TokenInfo, b: Balance.TokenInfo, wal
 
   return nameA.localeCompare(nameB, undefined, {sensitivity: 'base'})
 }
+
+export const getPriceImpactStatus = (priceImpact: number) => {
+  if (priceImpact < 1) return 'positive'
+  if (priceImpact > 10) return 'negative'
+  return 'warning'
+}
```

Comment:
@stackchain should we let this helper here or will be better to move it in the swap package?

Reply:
I'd say to leave it here, since it's a completely UI decision.